.circleci/config.yml

```diff
@@ -95,44 +95,17 @@ jobs:
     steps:
       - browser-tools/install-browser-tools
       - checkout
-      - run: "git clone https://github.com/bullet-train-co/bullet_train.git tmp/starter"
-
-      - run:
-          name: Link starter repository to the Ruby gem being tested.
-          command: "grep -v 'gem \"bullet_train-themes-light\"' tmp/starter/Gemfile > tmp/starter/Gemfile.tmp && mv tmp/starter/Gemfile.tmp tmp/starter/Gemfile && echo 'gem \"bullet_train-themes-light\", path: \"../..\"' >> tmp/starter/Gemfile"
 
       # TODO Figure out how to make these work for `tmp/starter`
       # - restore_cache: *restore_bundler_cache
       # - restore_cache: *restore_yarn_cache
 
-      # Install dependencies
-      - run: "cd tmp/starter && bundle install"
-      - run: "cd tmp/starter && bundle clean --force"
-      - run: "cd tmp/starter && bundle exec rake bt:link"
-      - run: "cd tmp/starter && yarn install"
-      - run: "cd tmp/starter && yarn build"
-      - run: "cd tmp/starter && yarn build:css"
-
       - *wait_for_docker
 
-      - run:
-          name: Run tests with Knapsack Pro
-          command: |
-            cd tmp/starter
-            export RAILS_ENV=test
-            SKIP_RESOLVE_TEST=1 bundle exec rails "knapsack_pro:queue:minitest[--verbose]"
-          environment:
-            KNAPSACK_PRO_CI_NODE_TOTAL: 16
-
-      # If you don't want to use Knapsack Pro, then use this configuration:
-      #
-      # - run:
-      #     name: Run unit tests
-      #     command: bundle exec rails test
-      # - run:
-      #     name: Run system tests
-      #     command: bundle exec rails test:system
-      #
+      - bullet_train_testing/starter_repo_minitest:
+          packageName: bullet_train-themes-light
+          knapsackPro: false
```

Comment:
@gazayas Do we want this to be `false`?

Reply:
Probably not :grimacing:
Changed in the latest commit! (The default value is `true`)